### PR TITLE
Add Farsi translation

### PR DESCRIPTION
**Pull request by v-ji**
Add the Farsi translation created by @Royaz28. We have tested the translation locally.

### CODE REVIEW

**Review by nalidnanac — APPROVED** [review]

File: src/i18n.ts [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+import { messages as faMessages } from "./locales/fa/messages"
+import { messages as frMessages } from "./locales/fr/messages"
+import { messages as jaMessages } from "./locales/ja/messages"
+import { messages as koMessages } from "./locales/ko/messages"
+import { messages as ruMessages } from "./locales/ru/messages"
+import { messages as ukMessages } from "./locales/uk/messages"
+import { messages as zhMessages } from "./locales/zh/messages"
+import { messages as tokMessages } from "./locales/jbo-tok/messages"
+
+const storageKey = "uiLanguage"
```
Comment (cmil) [inline src/i18n.ts:16]: Let's stick to the ending semicolons since that is the convention followed in the code base.

Comment (cmil) [reply src/i18n.ts]: > Is there a formatter config in this repo I have overlooked?

Not yet. I created #160 to fix that.

File: src/about.mdx [hunk 2]
```diff
@@ -114,6 +114,7 @@ In addition to English and German, we want to add more languages for our user in
 Many thanks to all translators who have contributed so far:
 
 * Chinese: [Xiaocui Qiu](https://www.dla-marbach.de/ueber-uns/mitarbeiterinnen-und-mitarbeiter/alle-mitarbeiterinnen-und-mitarbeiter/detail/14621/) (DLA Marbach)
+* Farsi: Roya Zendebudie (Freie Universität Berlin)
```
Comment (cmil) [inline src/about.mdx:117]: @lehkost added credits as well in the meantime. Please resolve the conflict.

Comment (v-ji) [reply src/about.mdx:117]: Done! :)

Comment (cmil) [reply src/about.mdx:117]: You intentionally lost the link?

Comment (cmil) [reply src/about.mdx:117]: Wait, there are two entries now. Only the linked one should remain I guess.

**Comment by cmil** [comment]
@v-ji let's not merge the main branch into feature branches. It makes for an ugly history. Rebase the feature branch instead. I'll do it.

**Pull request merged**